plugins/webp-uploads/settings.php

```diff
@@ -12,11 +12,25 @@
 }
 
 /**
- * Registers setting for generating both JPEG and WebP versions for image uploads.
+ * Registers setting for generating JPEG in addition to the selected modern format for image uploads.
  *
  * @since 1.0.0
+ * @since n.e.x.t The setting was made more general to cover outputting JPEG as a secondary type. The "webp" option naming
+ *                was left unchanged for backward compatibility.
+ * @since n.e.x.t The `perflab_modern_image_format` was added to enable selecting an output format.
+ *                Currently includes AVIF and WebP.
  */
 function webp_uploads_register_media_settings_field() {
+	register_setting(
+		'media',
+		'perflab_modern_image_format',
+		array(
```

Comment:
What about the `sanitize_callback`?

```suggestion
		array(
			'sanitize_callback' => static function ( $image_format ): string { return in_array( $image_format, array( 'webp', 'avif' ), true ) ? $image_format : 'webp'; },
```

Reply:
oh, nice!